packages/react-ui-components/src/ListPreviewElement/style.css

```diff
@@ -0,0 +1,23 @@
+.listPreviewElement {
+    composes: reset from './../reset.css';
+    background: var(--brandColorsContrastNeutral);
+    border-top: 1px solid var(--brandColorsContrastDarker);
+    cursor: pointer;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    font-size: 14px;
+
+    height: var(--goldenUnit);
+    line-height: var(--goldenUnit);
+    padding: 0 var(--spacing);
+    
+}
+
+.listPreviewElement--isHighlighted {
+        background: var(--brandColorsPrimaryBlue);
+}
+
+.listPreviewElement__icon {
+    composes: reset from './../reset.css';
+    padding-right: .5em;
```

Comment:
Don't we have a styling constant for this spacing? If we don't have the right size I think we should add one more.